docs/content/tools/xdebug.md

```diff
@@ -188,21 +188,21 @@ To debug Drush commands using Xdebug and VSCode, add the following to your path
     - Search for "php-debug"
     - Click "Settings" button below plugin
     - "Server" can be set to `*`
-    - "Server Listen Port" should be set to 9003
+    - "Server Listen Port" should be set to 9000
     - Make sure "Continue to listen for debug sessions even if the debugger windows are all closed" is checked. This will make the debugger window open automatically.
 
+## XDebug Modifications
+
+For versions of XDebug prior to v3.0.0, the following changes will need to be made to the projects `.docksal/docksal.yml`. To not introduce a breaking change within the CLI service and with IDEs the default port has been set back to 9000.
+
 ## Configuring Prior Versions
 
-For versions of XDebug prior to v3.0.0, the following changes will need to be made to the projects `.docksal/docksal.yml`. Additionally, to note the port has changed 
-from 9000 to 9003
+For versions of XDebug prior to v3.0.0, the following changes will need to be made to the projects `.docksal/docksal.yml`.
 
 ```
 services:
     cli:
         environment:
             - XDEBUG_CONFIG=remote_connect_back=0 remote_host=${DOCKSAL_HOST_IP}
```

Comment:
We can drop the `remote_connect_back=0` part from docs. It has been already dropped from services.yml

https://github.com/docksal/docksal/blob/b5aece64219dd12643dd4a315b6b68f6267d10be/stacks/services.yml#L144

See https://github.com/docksal/docksal/issues/1238#issuecomment-566779905